stdlib/auth/src/main/ballerina/src/auth/inbound_basic_auth_provider.bal

```diff
@@ -38,10 +38,14 @@ public type InboundBasicAuthProvider object {
         }
     }
 
-    # Attempts to authenticate with credentials.
-    #
-    # + credential - Credential
-    # + return - `true` if authentication is successful, otherwise `false` or `Error` occurred while extracting credentials
+# Attempts to authenticate with the given credentials.
```

Comment:
```suggestion
# Attempts to authenticate base64 encoded `username:password` credential.

Reply:
# Attempts to authenticate base64-encoded `username:password` credentials.